controllers/fingerprint.go

```diff
@@ -11,10 +11,63 @@ import (
 	"github.com/coduno/app/models"
 	"github.com/coduno/app/util"
 	"github.com/gorilla/mux"
+	"github.com/m4rw3r/uuid"
 )
 
+func CreateFingerprint(w http.ResponseWriter, r *http.Request, ctx context.Context){
+
+	var err error
+
+	if !util.CheckMethod(w,r, "POST"){
+		http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
+		return
+	}
+
+	firstName := mux.Vars(r)["firstName"]
+	lastName := mux.Vars(r)["lastName"]
+	email := mux.Vars(r)["email"]
+
+	coder := models.Coder{FirstName: firstName, LastName: lastName, Email: email}
+
+	coderKey, err := coder.Save(ctx)
+
+	if err != nil{
+		http.Error(w, "Internal server error: " + err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	challangeId := mux.Vars(r)["challangeId"]
+	challangeKey := datastore.NewKey(ctx, "challanges", challangeId, 0, nil)
+
+	token, err := uuid.V4()
+
+	if err != nil{
+		http.Error(w, "Internal server error: " + err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	fingerprint := models.Fingerprint{Coder: coderKey, Challenge: challangeKey, Token: token.String()}
+	_, err = fingerprint.Save(ctx)
+
+	if err != nil {
+		http.Error(w, "Internal server error: " + err.Error(), http.StatusInternalServerError)
+	}
+
+	response, err := json.Marshal(fingerprint)
+
+	if err != nil{
+		http.Error(w, "Json marshal error: "+err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	// TODO(pbochis):this is where we will send an e-mail to the candidate with smth like cod.uno/fingerprint/:token
```

Comment:
Please leave a space after `:` and start your sentence with an uppercase character.